src/content/toys.ts

```diff
@@ -32,17 +32,88 @@ function makeBlock() {
     overlaydiv.style.position = "fixed"
     overlaydiv.style.display = "block"
     overlaydiv.style.width = String(window.innerWidth)
-    overlaydiv.style.height = String(window.innerHeight)
-    overlaydiv.style.top = "0"
-    overlaydiv.style.left = "0"
-    overlaydiv.style.right = "0"
-    overlaydiv.style.bottom = "0"
-    overlaydiv.style.zIndex = "1000"
-    overlaydiv.style.opacity = "0.5"
+    overlaydiv.style.height = String(document.documentElement.scrollHeight)
+    overlaydiv.style.top = "0px"
+    overlaydiv.style.bottom = "0px"
+    overlaydiv.style.left = "0px"
+    overlaydiv.style.right = "0px"
+    overlaydiv.style.zIndex = "100"
```

Comment:
Why did you reduce the zIndex by a factor of 10? It used to be 1000

Reply:
That looks like a mistake. Should I revert?

Reply:
yes - `1000` is enough that stuff appears on top of most page elements. `100` might not be